File: sols/bootcamp_c/quest02/ex08/my_string_index.c
/*
**
** QWASAR.IO -- my_string_index
**
** @param {char*} param_1
** @param {char} param_2
**
** @return {int}
**
*/

int my_string_index(char* param_1, char param_2)
{
    int index=-1;
    for(int i=0; param_1[i] !='\0'; i++) 
    {
        if(param_1[i] == param_2)
        {
            index = i;
            break;
        }
    }

    return index;

}

File: sols/bootcamp_js/js-quest06/ex00/my_string_index.js
/*
**
** QWASAR.IO -- my_string_index
**
**
** @param {String} param_1
** @param {Character} param_2
** @return {integer}

**
*/


function my_string_index(param_1, param_2) {
    var index = -1;
    for(var i = 0; i < param_1.length; i++) {
        if (param_1[i] == param_2) {
            index = i;
            break;
        }
    }

    return index;
};
